### ISSUE TITLE
Cant deploy onto vultr: Invalid SSHKEYID

### ISSUE THREAD

**Issue by jtcressy** [pharmer/pharmer#345]
```
F1218 16:02:50.503678   18681 apply.go:34] Invalid SSHKEYID: 57dcbce7cd3b6
```
OS is fedora workstation 26. Tried deleting all resources and trying again, but it fails with the *same* invalid SSH KEY ID. It's almost as if this ID is hard-coded somewhere. I can't get it to deploy anything, and i'm certain the API key is correct because I can use the vultr cli to query the API

```
vultr sshkey list
```
Shows the list of SSH keys on my account but none of them match the ID above.

**Comment by tamalsaha** [comment]
Sounds like a duplicate of https://github.com/pharmer/pharmer/issues/318

**Comment by tamalsaha** [comment]
Fixed in #346
